docs/tutorial/tutorial.md

```diff
@@ -85,6 +85,8 @@ class ShoppingList extends React.Component {
 // Example usage: <ShoppingList name="Mark" />
 ```
 
+> You can import ```{ Component } from react``` and you can use only ```Component``` instead ```React.Component```;
```

Comment:
I don't think this is important in this tutorial (and it's not relevant for most people who would follow along on CodePen).